results/cmd/api/main_test.go

```diff
@@ -118,37 +119,287 @@ func TestUpdateTaskRun(t *testing.T) {
 		t.Fatalf("failed to create temp file for db: %v", err)
 	}
 
-	// Connect to fake server and create a taskrun
+	// Connect to fake server
 	ctx, cancel := context.WithTimeout(context.Background(), time.Second)
 	defer cancel()
-	objectmeta := &pb.ObjectMeta{Uid: "123459", Name: "mytaskrun", Namespace: "default"}
-	r, err := srv.CreateTaskRunResult(ctx, &pb.CreateTaskRunRequest{TaskRun: &pb.TaskRun{ApiVersion: "1", Metadata: objectmeta}})
-	if err != nil {
-		t.Fatalf("could not create taskrun: %v", err)
-	}
 
-	// Update the created taskrun
-	objectmeta = &pb.ObjectMeta{Uid: "123459", Name: "ziweifan", Namespace: "tomorrow"}
-	r, err = srv.UpdateTaskRunResult(ctx, &pb.UpdateTaskRunRequest{TaskRun: &pb.TaskRun{ApiVersion: "v1alpha1", Metadata: objectmeta}, ResultsId: r.GetResultsId()})
-	if err != nil {
-		t.Fatalf("could not update taskrun: %v", err)
-	}
-
-	// Validate by checking if we can get the updated taskrun
-	rows, err := srv.db.Query("SELECT taskrunlog FROM taskrun WHERE results_id = ?", r.GetResultsId())
-	if err != nil {
-		t.Fatalf("failed to query on database: %v", err)
+	// Validate by checking if ouput equlas expected TaskRun
```

Comment:
No changes needed here - just want to say this looks great and really thorough. Thanks! 😃

Reply:
done